sdk/appcenter/src/main/java/com/microsoft/appcenter/SessionContext.java

```diff
@@ -0,0 +1,220 @@
+package com.microsoft.appcenter;
+
+import android.support.annotation.VisibleForTesting;
+import android.support.annotation.WorkerThread;
+
+import com.microsoft.appcenter.utils.AppCenterLog;
+import com.microsoft.appcenter.utils.storage.StorageHelper;
+
+import java.util.LinkedHashSet;
+import java.util.Map;
+import java.util.NavigableMap;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.UUID;
+
+import static com.microsoft.appcenter.AppCenter.LOG_TAG;
+
+/**
+ * Persistent session history.
+ */
+public class SessionContext {
+
+    /**
+     * Key used in storage to persist sessions.
+     */
+    private static final String STORAGE_KEY = "sessions";
```

Comment:
Could you somewhere document (as a comment) the actual format of saved sessions?

Reply:
sure